### ISSUE TITLE
0.17.0-beta.4 Configuration file jspm.browser.js has been modified by another process

### ISSUE THREAD

**Issue by henrjk** [jspm/jspm-cli#1466]
I just created a new project and got this error with the latest jspm beta:

```
$ jspm init
Package.json file does not exist, create it? [Yes]: Yes
Init Mode (Quick, Standard, Custom) [Quick]: Custom
Prefix package.json properties under jspm? [Yes]: Yes
Local package name (recommended, optional): app
package.json directories.lib [src]: src
package.json directories.baseURL (optional): 
package.json directories.packages [jspm_packages]: jspm_packages
package.json configFiles.jspm [jspm.config.js]: jspm.config.js
package.json configFiles.jspm:browser [jspm.browser.js]: jspm.browser.js
System.config browser URL to src  [/src]: /src
System.config browser URL to jspm_packages [/jspm_packages]: /jspm_packages
System.config local package main: 
System.config local package format (esm, cjs, amd): esm
System.config transpiler (Babel, Traceur, TypeScript, None) [babel]: babel
     Updating registry cache...
ok   Installed dev dependency plugin-babel to npm:systemjs-plugin-babel@^0.0.2
     (0.0.2)

err  Error: Configuration file jspm.browser.js has been modified by another process.
    at ConfigFile.write (/Users/dev/code/experiment/jspm-17beta4/node_modules/jspm/lib/config/config-file.js:500:11)
    at SystemConfig.write (/Users/dev/code/experiment/jspm-17beta4/node_modules/jspm/lib/config/loader.js:308:13)
    at Object.exports.save (/Users/dev/code/experiment/jspm-17beta4/node_modules/jspm/lib/config/index.js:360:26)
    at /Users/dev/code/experiment/jspm-17beta4/node_modules/jspm/lib/install.js:1193:19
```

**Comment by guybedford** [comment]
Thanks, fixed in https://github.com/jspm/jspm-cli/commit/8aa29891ec1200c4b61dcc5a4805cc1a0b3ff15e.

**Comment by guybedford** [comment]
Released in 0.17.0-beta.5.
